internal/install/static_kubernetes_elastic_agent_yml.go

```diff
@@ -0,0 +1,125 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+const kubernetesDeployerElasticAgentYml = `---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kind-fleet-agent-clusterscope
```

Comment:
you are using `pod` dataset which actually queries from `kubelet`'s `stats/summary` endpoint. This one should be considered as node-scope since it will be performed on all nodes through the Daemonset. 
cluater-scope goes for `state_*` metricsets as well as `apiserver`.